### ISSUE TITLE
./install.py produces sudo error as well as announcing its installing rb2.97 instead of rb2.98

### ISSUE THREAD

**Issue by fossfreedom** [donaghhorgan/rhythmbox-plugins-rating-filters#4]
whilst testing RB2.98 on fedora 18, the following error was seen when running `./install.py`

![pic](http://i.imgur.com/daoGwPo.png)

I was also a little surprised to see "installing version 2.97" as well.

**Comment by fossfreedom** [comment]
however good news - on ubuntu 12.04 with RB2.97 it says it installs the RB2.97 version - and the sudo password was successfully prompted to enable you to type the password.

No further errors was observed in the terminal.

**Comment by donaghhorgan** [comment]
Great, thanks. It looks like my 2.98 test isn't working (I did a diff on the 2.97 and 2.98 binding docs to see what differences there were). I'll try something different so, and I might grab a copy of Fedora 18 tomorrow for future testing...

**Comment by donaghhorgan** [comment]
Right, I've got a few new VMs set up here and I can test on 2.96 through to 2.99.

It looks to me that the errors you got in the picture above are to do with the glib-compile-schemas command. I suspect the same thing would happen if you just ran 'sudo glib-compile-schemas /usr/share/glib-2.0/schemas/' in a terminal on your Fedora machine.

The installer got the version wrong though. I've updated the test code to something a little more elegant (just committed the changes), and it seems to be working for me here on 2.96-2.99 (all Ubuntu installs).
